### PR TITLE
Feature: xcframework

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

release this framework in XCFramework format

## Dependencies

Needs releases with:

- [ ] https://github.com/wireapp/wire-ios-shared-resources/pull/50
- [ ] https://github.com/wireapp/wire-ios-system/pull/59
- [ ] https://github.com/wireapp/wire-ios-testing/pull/37
- [ ] https://github.com/wireapp/wire-ios-utilities/pull/106
- [ ] https://github.com/wireapp/wire-ios-protos/pull/48
- [ ] https://github.com/wireapp/wire-ios-cryptobox/pull/54
- [ ] https://github.com/wireapp/wire-ios-transport/pull/190
- [ ] https://github.com/wireapp/wire-ios-images/pull/45
- [ ] https://github.com/wireapp/wire-ios-link-preview/pull/78
- [ ] https://github.com/wireapp/wire-ios-data-model/pull/1174

### CODE REVIEW

File: WireRequestStrategy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1429,6 +1403,10 @@
 			baseConfigurationReference = F1E47FE5207E0B72008D4299 /* ios-test-host.xcconfig */;
 			buildSettings = {
 				INFOPLIST_FILE = "$(SRCROOT)/Tests/Test Host/Info.plist";
+				LD_RUNPATH_SEARCH_PATHS = (
```
Comment (typfel) [inline WireRequestStrategy.xcodeproj/project.pbxproj:1406]: Should we define this in the xcconfig?

Comment (typfel) [reply WireRequestStrategy.xcodeproj/project.pbxproj:1406]: Will it be the same for all frameworks?

File: WireRequestStrategy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1429,6 +1403,10 @@
...
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
```
Comment (billypchan) [inline WireRequestStrategy.xcodeproj/project.pbxproj:1409]:
```suggestion
```

File: WireRequestStrategy.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1439,6 +1417,10 @@
...
```
Comment (billypchan) [inline WireRequestStrategy.xcodeproj/project.pbxproj:1423]:
```suggestion
```

File: WireRequestStrategy.xcodeproj/project.pbxproj (hunk 1 above)
Comment (billypchan) [reply WireRequestStrategy.xcodeproj/project.pbxproj]: yes, it is cleaned up and I will check other frameworks.

**Review by typfel — APPROVED** [review]

**Review by David-Henner — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**